CMakeLists.txt

```diff
@@ -34,7 +34,10 @@ endif()
 target_link_libraries(moonlight-common-c PRIVATE enet)
 
 if(MSVC)
-  target_compile_options(moonlight-common-c PRIVATE /W3 /wd4100 /wd4232 /wd5105 /WX)
+  target_compile_options(moonlight-common-c PRIVATE /W3 /wd4100 /wd4232 /wd5105)
+  if(NOT TARGET_UWP)
+	target_compile_options(moonlight-common-c PRIVATE /WX)
```

Comment:
What warnings are you seeing on UWP?

Reply:
It seems that it was a problem on some older builds and I kept it as-is. Reverting to upstream this file does not cause any problems, so I'm reverting this file